spec/models/redis_hit_counter_spec.rb

```diff
@@ -6,19 +6,19 @@
 
   describe ".current_timestamp" do
     it "returns the previous date at 2:59 AM UTC" do
-      Delorean.time_travel_to "2020/01/30 2:59 UTC" do
+      travel_to(Time.parse("2020-01-30 02:59:00 UTC")) do
```

Comment:
This an the other lines in the file don't need to manually call `Time.parse`. travel_to will convert the string to a Time by itself, so it should just be `travel_to "2020-01-30 02:59:00 UTC"`.